openlibrary/macros/SubjectTags.html

```diff
@@ -3,7 +3,7 @@
 $def render_subjects(label, subjects, track_value, prefix=""):
   $if subjects:
     <div class="section link-box">
-      <span class="clamp" data-before="+ ">
+      <span class="clamp" data-before="	&#9656; ">
```

Comment:
```suggestion
      <span class="clamp" data-before="&#9656; ">
```
Remove leading tab character from `data-before`.